app/src/main/java/com/bogdan/codeforceswatcher/components/compose/NoItemsView.kt

```diff
@@ -19,22 +20,26 @@ import com.bogdan.codeforceswatcher.components.compose.theme.AlgoismeTheme
 fun NoItemsView(
     @DrawableRes iconId: Int,
     @StringRes titleId: Int
-) = Column(
+) = LazyColumn(
```

Comment:
Hmm, I'm not against this change, but I'm interested in why you need to do it 🤔

Reply:
Swipe Refresh must contain scrolling content, so add `Modifier.scrollable` or `LazyColumn`. Otherwise, NoItemsView is'n refreshable.

Reply:
aaah, got it, thanks!